fontforgeexe/acorn2sfd.c

```diff
@@ -562,6 +562,8 @@ return;
 	    free(widths);
 	}
     }
+    if (m != 0)
+        free(mapping);
```

Comment:
It's not worth rewriting the pull request, but I consider it generally good form to set the pointer to NULL after freeing if the function is not terminating immediately. It makes detecting errors from use of freed memory easier to identify.